src/lib/components/quiz/quiz.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
+  import type { QuizData } from '$lib/constants/quiz';
+  import LeftArrow from '../icons/leftArrow.svelte';
+  import RightArrow from '../icons/rightArrow.svelte';
+  import MoreInfo from './moreInfo.svelte';
+
+  export let data: QuizData;
+
+  $: index = 0;
+  let response: string[] = [];
```

Comment:
We are currently using variables `tallyChoices` and `response` to keep track of the quiz score. Since this is a quiz, the quiz-taker is creating a list of responses. To keep the calculations simple, let's redeclare `response`.
```suggestion
  let responses: TeamMatch[] = [];
```
Note: Make sure `TeamMatch` is added to your import of `QuizData`.

Reply:
maybe also avoid team quiz specific naming so this can be reused for other quizzes. say for instance algo wanted to have a review quiz at the end of the whatever?

Reply:
Good thinking. We have a file of constants in [`/src/lib/constants/acm-paths.ts `](https://github.com/EthanThatOneKid/acmcsuf.com/blob/461c99785d165af72fd5125921ff2f6d252168a8/src/lib/constants/acm-paths.ts), but “paths” have been renamed to “teams”.

Reply:
Done